src/box/iproto.cc

```diff
@@ -1485,6 +1506,37 @@ iproto_connection_new(struct iproto_thread *iproto_thread)
 	return con;
 }
 
+/** Notify shutdown fiber that iproto thread shutdown is finished. */
+static void
+tx_process_shutdown(struct cmsg *m)
+{
+	struct iproto_thread *iproto_thread =
+		container_of(m, struct iproto_thread, shutdown_msg);
+	fiber_cond_signal(&iproto_thread->tx.shutdown_cond);
+}
+
+/** Check whether iproto thread shutdown is finished. */
+static void
+iproto_check_shutdown_finished(struct iproto_thread *iproto_thread)
+{
```

Comment:
Bad name: seems like this function just checks something and returns bool while in fact it initiates shutdown.

Reply:
Changed to the function which only sends message.
```c
/** Send message to TX thread to notify that connections drop is finished. */
static void
iproto_send_drop_finished(struct iproto_thread *iproto_thread)
```